app/src/main/res/values-ar/strings.xml

```diff
@@ -96,6 +96,7 @@
     <string name="lbl_specials">مقاطع خاصة</string>
     <string name="lbl_chapters">الأجزاء</string>
     <string name="lbl_cast_crew">الممثلون</string>
+    <string name="lbl_additional_parts">أدوار إضافية</string>
```

Comment:
Only english strings (values/strings.xml) should be added in git. Everything else should be done in Weblate (translate.jellyfin.org).

(Make sure this doesn't stay in the git history to prevent breakage)